apps/consulting/utils/getArticlesPaths/isSlugForPost.ts

```diff
@@ -0,0 +1,4 @@
+import { ARTICLES_DIRECTORY_SLUG } from './constants';
+
+export const isSlugForPost = (slug: string): boolean =>
+  slug.startsWith(ARTICLES_DIRECTORY_SLUG);
```

Comment:
Interesting. I kind of would think that this would fix this:

<img width="968" alt="image" src="https://user-images.githubusercontent.com/11325439/217961759-65751154-cb2a-4baf-9a36-656469ada39e.png">

But I guess not.

Reply:
It doesn't fix that issue because this function isn't used in `getPaths()`. I started to slip that fix into this PR but then it felt like I was putting too much into one PR. I will add a comment below showing you in the code how this can be fixed.